packages/java/src/browser/java-client-contribution.ts

```diff
@@ -44,9 +48,23 @@ export class JavaClientContribution extends BaseLanguageClientContribution {
 
     protected onReady(languageClient: ILanguageClient): void {
         languageClient.onNotification(ActionableNotification.type, this.showActionableMessage.bind(this));
+        languageClient.onNotification(StatusNotification.type, this.showStatusMessage.bind(this));
         super.onReady(languageClient);
     }
 
+    protected showStatusMessage(message: StatusReport) {
+        window.clearTimeout(this.statusBarTimeout);
```

Comment:
1: it's not really necessary her, but it doesn't hurt either to check if `this.statusBarTimeout` is undefined.
2: also please set `this.statusBarTimeout` to undefined when removing the statusbar element.